developer_guides/debugability/probes/index.rst

```diff
@@ -17,7 +17,28 @@ Requirements
 Enabling Probes
 ***************
 
-- Enable the Linux kernel configuration option: ``DEBUG_FS``
+- Enable the following Linux kernel configuration options:
+
+	.. code-block:: bash
+
+           CONFIG_DEBUG_FS=y
+           CONFIG_SND_SOC_SOF_DEBUG_PROBES=y
+           CONFIG_SND_SOC_SOF_HDA_PROBES=y
+
+- Add Probe compress CPU DAI BE to the dai links of the machine driver under use.
+  Example of the cpu and codec DAI entries:
+
+	.. code-block:: bash
+
+           #ifdef CONFIG_SND_SOC_SOF_DEBUG_PROBES
+           SND_SOC_DAILINK_DEF(compress_pin,
+           DAILINK_COMP_ARRAY(COMP_CPU("Probe Extraction CPU DAI")));
+           SND_SOC_DAILINK_DEF(dummy,
+           DAILINK_COMP_ARRAY(COMP_DUMMY()));
+           #endif
```

Comment:
I feel like this code block could be better indented. Also wrong language for highlighting, shouldn't it be c instead of bash?

Reply:
Ack, will update, thanks for your review!!

Reply:
no, that's not right. We don't want the probes in the machine drivers, they should be in a separate SOF client - @fredoh9 and @ranj063 were working on this.

Reply:
Since the probes features is already included in the JSL SOF v0.3 release, I feel it's good to provide the current way that works and update later when multi-client comes.

Reply:
the machine driver changes are not and will not be contributed upstream, so I don't see the point of documenting a point solution that doesn't reflect the desired direction?

Reply:
Sure, I can remove the machine driver change. Thanks!!